css/components/cards.css

```diff
@@ -0,0 +1,121 @@
+/* Card Container */
+.card {
+  max-width: 100%;
+  width: 22rem;
+  position: relative;
+  background-color: var(--text-white);
+  border-radius: var(--radius-md);
+  border: 1px solid var(--color-grey-300);
+  align-self: flex-start;
+}
+
+.card-image {
+  display: block;
+  width: 100%;
+  margin: auto;
+  border-radius: var(--radius-md);
+  overflow: hidden;
+  cursor: pointer;
+}
+
+.card-body,
+.card-buttons,
+.card-price,
+.card-content {
+  padding: var(--size-000) var(--size-0);
+}
+
+/* card icons*/
+
+.card-icons {
+  position: absolute;
+}
+
+.card-icons > button {
+  background-color: var(--color-grey-300);
+  padding: var(--size-000);
+  width: var(--size-4);
+  height: var(--size-4);
+  border-radius: var(--radius-round);
+
+  opacity: 0.6;
+  cursor: pointer;
+}
+
+.card-icons .icon {
+  font-size: var(--size-0);
```

Comment:
Why have a size 0?

Reply:
It's not size 0 but 1rem.

It maybe needs to be more thought out in the future but right now I needed these naming to better remember and not go back again and again to declarations to check, that's why.